developers/envvarsetup.py

```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+
+if len(sys.argv) < 2:
+    print("Usage: envarsetup.py [full path to parent directory of sleuthkit, autopsy, etc.]")
+    exit(1)
+
+'''
+The following 7 lines can be configured to the specified paths (if different) on your system.
+open_jdk_64_home is the 64 bit jdk and is the assumed default
+source_base_path is the directory containing all necessary repos (i.e. autopsy, sleuthkit, etc.)
+open_jdk_32_home and postgres_32_home are only necessary if building binaries
+'''
+source_base_path = sys.argv[1]
+open_jdk_64_home = "C:\\Program Files\\ojdkbuild\\java-1.8.0-openjdk-1.8.0.222-1"
+postgres_home = "C:\\Program Files\\PostgreSQL\\9.5"
+ant_home = "C:\\Program Files\\NetBeans 8.2\\extide\\ant"
+open_jdk_32_home = "C:\\Program Files (x86)\\ojdkbuild\\java-1.8.0-openjdk-1.8.0.222-1"
+postgres_32_home = "C:\\Program Files (x86)\\PostgreSQL\\9.5"
+open_jdk_home = open_jdk_64_home
+
+
+sep = '\\'
+path_env_sep = ";"
+
+
+def set_var(env_var, env_val):
+    print("Setting {0} to {1}".format(env_var,env_val))
+    os.system("SETX {0} \"{1}\" /M".format(env_var,env_val))
+
+
+def add_path(paths):
+    print("Adding to path: {0}".format(paths))
+    # work around for command prompt to access PATH via %PATH%
+    cmd = "cmd.exe /k SETX PATH \"%PATH%;{0}\" /M & exit".format(path_env_sep.join(paths))
+    os.system(cmd)
+
+
+set_var("JAVA_HOME", open_jdk_home)
```

Comment:
All uses of foo + sep + bar can be replaced with os.path.join which will handle formatting paths:
![python_escape](https://user-images.githubusercontent.com/41580268/74756876-3bee9780-5243-11ea-99ec-12a527974853.PNG)
![python_windows](https://user-images.githubusercontent.com/41580268/74756883-3ee98800-5243-11ea-9bd5-aa5e1e7b9846.PNG)